korge/src/commonMain/kotlin/korlibs/korge/Korge.kt

```diff
@@ -60,6 +60,7 @@ suspend fun Korge(
     trace: Boolean = false,
     context: Any? = null,
     fullscreen: Boolean? = null,
+    undecorated: Boolean? = null,
```

Comment:
Can you create a class called `WindowConfig`, initially with `data class WindowConfig(val undecorated: Boolean = false)`, so we can add there other stuff without polluting there method more.

Full proposal in the comment.